crates/proto/src/domain/notes.rs

```diff
@@ -1,17 +1,21 @@
 use miden_objects::{
-    notes::{NoteMetadata, NoteTag, NoteType},
+    crypto::hash::rpo::RpoDigest,
+    notes::{NoteId, NoteInclusionProof, NoteMetadata, NoteTag, NoteType},
     Felt,
 };
 
-use crate::errors::{ConversionError, MissingFieldHelper};
+use crate::{
+    errors::{ConversionError, MissingFieldHelper},
+    generated::note::{NoteInclusionProof as NoteInclusionProofPb, NoteMetadata as NoteMetadataPb},
+};
```

Comment:
nit (and not for this PR): I think it would be cleaner to add `Proto` suffix instead of `Pb` to the aliases. For example, `NoteMetadata as NoteMetadataProto`.

Reply:
I'll see if it doesn't make more sense to alias the module instead

```rust
use crate::generated::note as proto;

impl From<T> for proto::T {...}
```

Reply:
Ended up using `generated` for the most part.

Reply:
I actually prefer adding type aliases as I think it makes the code a bit easier to read - e.g., instead of `generated::merkle::MerklePath` we get `MerklePathProto`. But let's keep things as they currently are in this PR.